README.md

```diff
@@ -103,6 +103,14 @@ Options:
   -h, --help                display help for command
 ```
 
+### npm executable
+
+By default, this tool will use global `npm`. This can  be changed by setting environmental variable `npm_execpath` to path of `npm-cli.js` executable, for example:
+
+```shell
```

Comment:
please change the fenced code language from `shell` to `bash`.
the example might not work in all shells.